### PR TITLE
Release notes for week of 8/31 - 9/4

### PR DESCRIPTION
**Pull request by srpearce**
@lamagnifica  Here's the only item in the release notes so far. Let me know if you want to add any LTI info.

(Changed @catong to @lamagnifica above - I'd looked at the wrong line on the release notes rotation page)

### CODE REVIEW

File: en_us/release_notes/source/2015/09-02-2015.rst [hunk 1]
```diff
@@ -0,0 +1,25 @@
+###################################
+2 September 2015
+###################################
+
+The following information reflects what is new in the edX Platform during the
```
Comment (catong) [inline en_us/release_notes/source/2015/09-02-2015.rst]: "... what is new during the week of" seems odd: "what is new... for the week of" ?

File: en_us/release_notes/source/2015/studio/studio_0902_2015.rst [hunk 2]
```diff
@@ -0,0 +1,6 @@
+
+When course authors included HTML tags inside a ``<choicehint>`` or
+``<demandhint>`` tag in a problem, the content inside the HTML tags and any
+following text was invisible to students. This problem has been fixed.
```
Comment (catong) [inline en_us/release_notes/source/2015/studio/studio_0902_2015.rst]: @srpearce I'm not entirely clear what "any following text" means. Does this mean that in addition to anything within `<choicehint>` or `<demandhint>` tags not displaying, any HTML _after_ an occurrence of either tag wasn't displaying?

**Comment by lamagnifica** [comment]
I'm changing this week's reviewer to @catong since she's on it!

**Comment by srpearce** [comment]
@catong I found another item! See the LMS page. Also, let me know if the Studio item makes more sense.

**Comment by catong** [comment]
@srpearce Yay for the added LMS item, which is a nice addition. The text describing the HTML fix is much clearer to me now, thanks. LGTM!

**Comment by srpearce** [comment]
@explorerleslie, can you give these two items a quick review?

File: en_us/release_notes/source/2015/studio/studio_0902_2015.rst [hunk 3]
```diff
@@ -0,0 +1,5 @@
+When course authors included HTML tags inside a ``<choicehint>`` or
+``<demandhint>`` tag in a problem, the content inside the HTML tags and any
+content after the HTML tags was invisible to learners. This problem has been
+fixed. (TNL-3096)
```
Comment (catong) [inline en_us/release_notes/source/2015/studio/studio_0902_2015.rst]: @srpearce A couple of minor nits: make things plural, and add "XML" for clarity.

"... within `<choicehint>` or `<demandhint>` XML tags in problems, the content inside..."

(And I changed the first "inside" to "within" for variation.)

**Comment by explorerleslie** [comment]
:+1:

**Pull request merged**